tests/gui/control/test_progress.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from taipy.gui import Gui
+
+
+def test_circular_progress_indeterminate_md(gui: Gui, helpers):
+    md_string = "<|progress|>"
+    expected_list = ["<Progress"]
+    helpers.test_control_md(gui, md_string, expected_list)
+
+
+def test_circular_progress_determinate_md(gui: Gui, helpers):
+    md_string = "<|progress|value=50|showValue=True|>"
+    expected_list = ["<Progress", 'value="50"']
```

Comment:
I thought it would be "defaultValue={50}" ?

Reply:
Should add "showValue={true}"

Reply:
the `showValue=True` param is already present here?
Also, could you please guide about the `defaultValue`, like without `defaultValue` when I run the test, it's passing. Is it necessary to add defaultValue=50 praram?

Reply:
I'll add if it's required 👍

Reply:
Actually, by using `defaultValue={50}`, the tests is failing, should I keep it as it is?

Reply:
Ahh! Sorry, I by mistake changed the `value` praam to `defaultValue` instead of adding another `defaultValue` param. It's working now. Both `showValue` and `defaultValue` params are working 👍